### ISSUE TITLE
The '%u' template variable is used for both URL and updated time

### ISSUE THREAD

**Issue by azenk** [andjosh/clubhouse-cli#137]
### Expected Behavior

It should be possible to use a template variable to print the updated timestamp. 

### Actual Behavior

The `%u` variable is used for both URL and updated time. The URL substitution is performed first, so it's impossible to print the time that a story was updated at.

https://github.com/andjosh/clubhouse-cli/blob/62ea8661756eea4b6d826d54fbf94b2e1380b347/src/lib/stories.ts#L315-L317

### Steps to Reproduce

Run a club search command using the `%u` template variable. The expected result in this case is to have a column with the creation timestamp and a second with the updated time.  The actual result is creation timestamp and the URL.

```bash
club search 'completed:2020-01-19..2020-01-22' -f '%c %u %s'
```

**Comment by andjosh** [comment]
Very true! I will fix that.

**Comment by andjosh** [comment]
Published in [v2.2.1](https://github.com/andjosh/clubhouse-cli/blob/master/CHANGELOG.md#221---2020-01-23)
